webrtc.html

```diff
@@ -2521,19 +2521,6 @@ <h4>
                                       <var>transceiver</var>.{{RTCRtpTransceiver/[[Sender]]}}.{{RTCRtpSender/[[SendEncodings]]}}.
                                     </p>
                                   </li>
-                                  <li>
-                                    <p>
-                                      Update the paused status as indicated by
-                                      [[RFC8853]] of each simulcast
-                                      layer by setting the
-                                      {{RTCRtpEncodingParameters/active}}
-                                      member on the corresponding dictionaries
-                                      in
-                                      <var>transceiver</var>.{{RTCRtpTransceiver/[[Sender]]}}.{{RTCRtpSender/[[SendEncodings]]}}
-                                      to <code>true</code> for unpaused or to
-                                      <code>false</code> for paused.
-                                    </p>
```

Comment:
This removal is good. Though as mentioned by @Orphis in the meeting I wonder if it is sufficient.

Do we perhaps also need to say somewhere something like _"For offers and answers [[RFC7728]] is not supported."_ and make it a non-normative(?) reference. Perhaps in 5.4.1 after _"Alternatives and restrictions (described in [RFC8853]) are not supported."_? 

The reason I wonder this is that the algorithm steps here happen on main thread *after* the description has been generated in the background, and is mostly preoccupied with updating [[SendEncodings]] which (I think??) is mostly used to hold the JS observable states (on main thread) to match what is mostly described in IETF specs, and not actually used anywhere to control what actually gets paused or not in the back end AFAICT (though I could be wrong about this).

Reply:
I volunteer to add this in a follow-up to unblock this.